libcloudforensics/providers/gcp/forensics.py

```diff
@@ -357,3 +357,52 @@ def InstanceNetworkQuarantine(project_id: str,
         'gcloud compute ssh --zone "{0:s}" "{1:s}" --project "{2:s}"\n'
         'Connecting from the browser via GCP console will not work.'.format(
               instance.zone, instance_name, project_id))
+
+def VMRemoveServiceAccount(project_id: str,
+                           instance_name: str,
+                           leave_stopped: bool = False) -> bool:
+  """
+  Remove a service account attachment from a GCP VM
+
+  Service account attachments to VMs allow the VM to obtain credentials
+  via the instance metadata service to perform API actions. Removing
+  the service account attachment will prevent credentials being issued.
+
+  Note that the instance must be powered down for this action.
+
+  Args:
+    project_id (str): Google Cloud Project ID.
+    instance_name (str): The name of the virtual machine.
+    leave_stopped (bool): Optional. True to leave the machine powered off.
+  Returns:
+    success (bool)
```

Comment:
It's better to be a bit more explicit in the conditions in which booleans are returned:

```suggestion
    bool: True if the service account was successfully removed, False otherwise.
```